qa-pipelines/tasks/cf-deploy-upgrade-common.sh

```diff
@@ -153,7 +158,9 @@ set -o allexport
 # The external_ip is set to the internal ip of a worker node. When running on openstack or azure, 
 # the public IP (used for DOMAIN) will be taken from the floating IP or load balancer IP.
 external_ip=$(kubectl get configmap -n kube-system cap-values -o json | jq -r '.data["internal-ip"]')
-public_ip=$(kubectl get configmap -n kube-system cap-values -o json | jq -r '.data["public-ip"] // empty')
+public_ip=$(kubectl get configmap -n kube-system cap-values -o json | jq -r '.data["public-ip"]')
+garden_rootfs_driver=$(kubectl get configmap -n kube-system cap-values -o json | jq -r '.data["garden-rootfs-driver"] // "btrfs"')
+garden_use_btrfs=$([[ ${garden_rootfs_driver} == btrfs ]] && echo true || echo false)
```

Comment:
don't need this

Reply:
We still need this to set both variables correctly; with the new commit, we're setting both regardless of the chart version, as having both set won't hurt and will allow our pipeline to continue working with future versions as well as builds from develop (as Andreas noted, support for the new variable is required in those situations).

Reply:
https://github.com/SUSE/scf/blob/develop/container-host-files/etc/scf/config/role-manifest.yml#L2670n empty value means BTRFS is used. Set to "false" to use overlay-xfs.